llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -399,6 +399,12 @@ static cl::opt<bool> EnableEarlyExitVectorization(
     cl::desc(
         "Enable vectorization of early exit loops with uncountable exits."));
 
+static cl::opt<bool>
+    EnableEarlyExitWithFFLoads("enable-early-exit-with-ffload", cl::init(false),
```

Comment:
In the spirit of [incremental development](https://llvm.org/docs/DeveloperPolicy.html#id24) can we remove this option just have it on by default?